docs/upgrading/remix.md

```diff
@@ -138,9 +263,15 @@ hydrateRoot(
 );
 ```
 
-### Step 7 - Update types for `AppLoadContext`
+## 8. Update types for `AppLoadContext`
+
+<docs-info>
+
+If you were using `remix-serve` you can skip this step. This is only applicable if you were using a custom server in Remix v2.
 
-<docs-info>If you were using `remix-serve` you can skip this step. This is only applicable if you were using a custom server in Remix v2.</docs-info>
+Additionally, if you used the codemod you can skip this step as it was automatically completed.
```

Comment:
Codemod doesn't handle updating AppLoadContext types.

Reply:
Whoops! Thanks for pointing that out, I'll remove it